addon-api/content-script/Tab.js

```diff
@@ -613,6 +613,11 @@ export default class Tab extends Listenable {
         from: () => [],
         until: () => [],
       },
+      afterProfileCountry: {
+        element: () => q("p.profile-details > .location"),
+        from: () => [],
+        until: () => [],
+      },
```

Comment:
I'm not sure how exactly this would work, as the `<span class="location">` already has a text node inside it (the user's country) 🤔
Maybe this space should create its own div and the space should be inside that new element?

Reply:
Self reminder: some other addons might be caused by changing the location CSS, such as `better-featured-project`, `scratchr2`

Reply:
`scratchr2` only changes minor styling such as color and shadow, but it does change the line height from `1.5385em` to `16px`, which doesn't make a difference.

`better-featured-project` adds `display: block !important;` to the element, which moves the new shared space element below the location. That's something that needs to be fixed.

I looked through the other addons. No other ones seem to affect the element, with the exception of the dark mode addon, but that just changes the colors.